taskcluster/ci/nimbledroid/kind.yml

```diff
@@ -18,9 +19,17 @@ job-defaults:
         using: run-commands
         use-caches: false
         secrets:
-            - name: project/mobile/fenix/nimbledroid
-              key: api_key
-              path: .nimbledroid_token
+            by-level:
+                '3':
+                    - name: project/mobile/fenix/nimbledroid
+                      key: api_key
+                      path: .nimbledroid_token
+                default: []
+        pre-commands:
+            by-level:
+              '3': []
+              default:
+                - [echo, '--', '>', .nimbledroid_token]
```

Comment:
This line is inspired by this logic in the build tasks: https://github.com/mozilla-mobile/fenix/blob/b23ee38082698ad711361fb9dd25465e647efb83/taskcluster/fenix_taskgraph/transforms/build.py#L52-L59